flow360/component/simulation/meshing_param/params.py

```diff
@@ -29,6 +31,38 @@
 ]
 
 
+class MeshingDefaults(Flow360BaseModel):
+    """
+    Default/global settings for meshing parameters.
+    """
+
+    ##::   Default surface edge settings
+    surface_edge_growth_rate: float = pd.Field(
+        1.2, ge=1, description="Global growth rate of the anisotropic layers grown from the edges."
+    )
+
+    ##::    Default boundary layer settings
+    boundary_layer_growth_rate: float = pd.Field(
+        1.2, description="Default growth rate for volume prism layers.", ge=1
+    )
+    # pylint: disable=no-member
+    boundary_layer_first_layer_thickness: Optional[LengthType.Positive] = pd.Field(
+        None, description="Default first layer thickness for volumetric anisotropic layers."
+    )  # Truly optional if all BL faces already have first_layer_thickness
+
+    ##::    Default surface layer settings
+    surface_max_edge_length: Optional[LengthType.Positive] = pd.Field(
+        None, description="Default maximum edge length for surface cells."
+    )  # Not actually optional and will raise error (TODO) if surface meshing is needed
```

Comment:
I think these should be required. Otherwise, user will have two methods of providing this information, one is by MeshingDefaults, the other is by Surface refinement and BL refinement

Reply:
I think we no longer support (in meshing) specifying global values using a None entityList.
Also we cannot make this required if user project starts from SM (in the future) and only want to construct VM.